policy/pa.go

```diff
@@ -174,24 +174,24 @@ func isDNSCharacter(ch byte) bool {
 
 var (
 	errInvalidIdentifier    = berrors.MalformedError("Invalid identifier type")
-	errNonPublic            = berrors.MalformedError("Name does not end in a public suffix")
-	errICANNTLD             = berrors.MalformedError("Name is an ICANN TLD")
-	errPolicyForbidden      = berrors.RejectedIdentifierError("Policy forbids issuing for name")
-	errInvalidDNSCharacter  = berrors.MalformedError("Invalid character in DNS name")
-	errNameTooLong          = berrors.MalformedError("DNS name too long")
-	errIPAddress            = berrors.MalformedError("Issuance for IP addresses not supported")
-	errTooManyLabels        = berrors.MalformedError("DNS name has too many labels")
-	errEmptyName            = berrors.MalformedError("DNS name was empty")
-	errNameEndsInDot        = berrors.MalformedError("DNS name ends in a period")
-	errTooFewLabels         = berrors.MalformedError("DNS name does not have enough labels")
-	errLabelTooShort        = berrors.MalformedError("DNS label is too short")
-	errLabelTooLong         = berrors.MalformedError("DNS label is too long")
-	errMalformedIDN         = berrors.MalformedError("DNS label contains malformed punycode")
-	errInvalidRLDH          = berrors.RejectedIdentifierError("DNS name contains a R-LDH label")
-	errTooManyWildcards     = berrors.MalformedError("DNS name had more than one wildcard")
-	errMalformedWildcard    = berrors.MalformedError("DNS name had a malformed wildcard label")
-	errICANNTLDWildcard     = berrors.MalformedError("DNS name was a wildcard for an ICANN TLD")
-	errWildcardNotSupported = berrors.MalformedError("Wildcard names not supported")
+	errNonPublic            = berrors.MalformedError("Domain name does not end with a valid public suffix (TLD)")
+	errICANNTLD             = berrors.MalformedError("Domain name is an ICANN TLD. Certificate can not be issued")
+	errPolicyForbidden      = berrors.RejectedIdentifierError("The ACME server can not issue a certificate for this domain name, because part of the name is forbidden by policy.")
+	errInvalidDNSCharacter  = berrors.MalformedError("Domain name contains an invalid character")
+	errNameTooLong          = berrors.MalformedError("Domain name is longer than 230 bytes, which is the maximum length supported by this server.")
```

Comment:
inconsistent period at the end of the error

Reply:
yep